packages/customize-widgets/src/components/welcome-guide/index.js

```diff
@@ -43,8 +43,7 @@ export default function WelcomeGuide( { sidebar } ) {
 					  ) }
 			</p>
 			<Button
-				// TODO: Switch to `true` (40px size) if possible
-				__next40pxDefaultSize={ false }
+				__next40pxDefaultSize
```

Comment:
- Active an non-blocks based theme
- Visit customizer, then widgets. The guide will display on first visit

![Screenshot 2024-10-01 at 21 50 56](https://github.com/user-attachments/assets/bfda7244-d945-4c6f-a384-cfa210095103)

Reply:
Similarly, to be consistent with the sizing of the other buttons, I'd personally prefer `size="compact"` for this one.